packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1577,23 +1723,53 @@ type PartialArgsType<Adapter> = Adapter extends CallAdapter
   ? Partial<AzureCommunicationCallAdapterArgs>
   : Partial<TeamsCallAdapterArgs>;
 
+type PartialArgsOutboundType<Adapter> = Adapter extends CallAdapter
+  ? Partial<AzureCommunicationOutboundCallAdapterArgs>
+  : Partial<TeamsCallAdapterArgs>;
+
 type AdapterOf<AdapterKind extends 'AzureCommunication' | 'Teams'> = AdapterKind extends 'AzureCommunication'
   ? CallAdapter
   : never | /* @conditional-compile-remove(teams-identity-support) */ TeamsCallAdapter;
 
 /**
  * @private
  */
-const useAzureCommunicationCallAdapterGeneric = <
+function useAzureCommunicationCallAdapterGeneric<
   AdapterKind extends 'AzureCommunication' | 'Teams',
```

Comment:
'AzureCommunication' | 'Teams' can be extracted out as it's own type. Used more than twice in this file.

Reply:
Discussed it further, there's only three instances used as of now. 
We can look into typing this out once cte is released.